packages/wdio-junit-reporter/src/index.ts

```diff
@@ -149,9 +149,9 @@ class JunitReporter extends WDIOReporter {
         specFileName: string,
         suite: SuiteStats
     ) {
-        const suiteName = this._prepareName(suite.title)
+        //const suiteName =
         const filePath = specFileName.replace(process.cwd(), '.')
-
+        const suiteName = this.options.testSuiteName ? this._prepareName(this.options.testSuiteName({ name: this.options.testSuiteName.name, suite })) : this._prepareName(suite.title)
```

Comment:
What is `this.options.testSuiteName` here? A function or an object with the property `name`?

Reply:
Is the function defined before

Reply:
Should we stay consistent and call the second argument `activeSuiteName`?

```suggestion
        const suiteName = this.options.testSuiteName
            ? this.options.testSuiteName({
                name: this.options.testSuiteName.name,
                suite
             })
             : this._prepareName(suite.title)
```